Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM  python:3.6.9-alpine3.10
```

Comment:
Please `s/  / /` (remove the extra space) and use `3.6-alpine` tag to automatically use the latest version.